unlock-app/src/stories/content/AccountContent.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Provider } from 'react-redux'
+import { storiesOf } from '@storybook/react'
+import { select, withKnobs } from '@storybook/addon-knobs'
+import createUnlockStore from '../../createUnlockStore'
+import { ConfigContext } from '../../utils/withConfig'
+import AccountContent from '../../components/content/AccountContent'
+
+const baseState = {
+  account: {},
+}
+const loggedInState = {
+  account: {
+    emailAddress: 'jenny@googlemail.com',
+  },
+}
+const loggedInWithCards = {
+  account: {
+    emailAddress: 'jenny@googlemail.com',
+    cards: [{ id: 'card1' }],
+  },
+}
+
+const config = {
+  stripeApiKey: 'pk_this_is_not_a_real_key',
+}
+
+storiesOf('AccountContent (iframe embed for paywall)', module)
+  .addDecorator(withKnobs)
+  .add('The embed', () => {
+    const label = 'Component State'
+    const options = {
```

Comment:
really clever use of knobs

Reply:
would also be useful to have separate stories for each state, so chromatic will catch any changes to them

Reply:
Yup, that's something I plan to do once the "real" components are in. We'll have one for everything but the Stripe one.